cli/cmd/install_test.go

```diff
@@ -125,6 +125,7 @@ func TestRender(t *testing.T) {
 		ProxyInjector:      defaultValues.ProxyInjector,
 		ProfileValidator:   defaultValues.ProfileValidator,
 		Tap:                defaultValues.Tap,
+		SmiMetrics:         defaultValues.SmiMetrics,
```

Comment:
Here and elsewhere: prefer _SMI_ over _Smi_. See https://github.com/golang/go/wiki/CodeReviewComments#initialisms.